test/AuthorizationHeaderTest.php

```diff
@@ -104,34 +112,54 @@ public function testCaseInsensitiveRequestMethod()
 
     /**
      * Ensures an exception is thrown if a request does not have an Authorization header.
-     *
-     * @expectedException \Acquia\Hmac\Exception\MalformedRequestException
      */
     public function testCreateFromRequestNoAuthorizationHeader()
     {
+        $this->setExpectedException(
+            '\Acquia\Hmac\Exception\MalformedRequestException',
```

Comment:
Assuming `setExpectedException()` is necessary, you can use the `::class` name resolution operator here: we depend on PHP 5.5.